packages/app-admin/src/components/FileManager/FileDetails.tsx

```diff
@@ -149,7 +150,17 @@ type FileDetailsProps = {
 };
 export default function FileDetails(props: FileDetailsProps) {
     const { file, uploadFile, validateFiles } = props;
-    const isImage = file.name.match(/.(jpg|jpeg|png|gif)$/i)
+
+    const isImage = file => {
```

Comment:
Good work @imrishabh18 👍 

I would just move this function outside of React component because it doesn't depend upon it.
Rest looks good.

Reply:
Okay, Can you please tell how to figure out which functions to put outside of the functional component and when to define it inside the component

Reply:
If a function can be standalone, meaning - it doesn't need anything but its parameters to work, you can move it outside of a component. That way, React component will be always calling the same function by reference and won't be re-creating the function on each rerender.